app/ide-desktop/lib/dashboard/src/hooks/eventCallback.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
nit: this file should probably be called `eventCallbackHooks.ts` for consistency (same with the other new hooks)

Reply:
> yet this isn't happening in this codebase

Not happening _YET_, the project is it's early days.

Reply:
yes, but we don't have umbrella files *at all*. we have files *with a suffix*, but that is not the same as having a file called *just* `hooks.ts`

Reply:
> not sure about this either. perhaps you mean redundant

Yea, sorry, tough morning :)

Reply:
> I've seen enough projects in my life fallen under the pressure of this approach

fair, but i'm still not sure whether choosing a specific naming convention over another one, will make this more likely to happen